diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/background.js ./background.js
--- ../Fraud-Blocker-multi-browser-extension/background.js	2017-03-10 13:41:19.487672589 +0100
+++ ./background.js	2017-03-10 13:48:18.963654966 +0100
@@ -4,13 +4,17 @@
 
 function checkTab(info){
   chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
-    var res = getStatus(tabs[0].url);
-    if(res.msg=="grey"){
-      var nFraud=res.nFraud;
-      if(isNaN(nFraud)) //sanitize
-        nFraud='error';
-      chrome.browserAction.setBadgeText({tabId: tabs[0].tabId, text: nFraud});
-      chrome.browserAction.setBadgeBackgroundColor({tabId: tabs[0].tabId, color: "#cc0000"});
+    if(tabs[0].url.substring(0,9)!="chrome://"){      
+      var res = getStatus(tabs[0].url);
+      if(res.msg=="grey"){
+        var nFraud=res.nFraud;
+        if(isNaN(nFraud)) //sanitize
+          nFraud='error';
+        chrome.browserAction.setBadgeText({tabId: tabs[0].tabId, text: nFraud});
+        chrome.browserAction.setBadgeBackgroundColor({tabId: tabs[0].tabId, color: "#cc0000"});
+      }
+      else
+        chrome.browserAction.setBadgeText({tabId: tabs[0].tabId, text: ""});
     }
     else
       chrome.browserAction.setBadgeText({tabId: tabs[0].tabId, text: ""});
Solo in ../Fraud-Blocker-multi-browser-extension: background.js.BACK
diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/manifest.json ./manifest.json
--- ../Fraud-Blocker-multi-browser-extension/manifest.json	2017-03-10 13:50:36.603649184 +0100
+++ ./manifest.json	2017-03-10 13:50:56.863648333 +0100
@@ -4,19 +4,14 @@
   "name": "Fraud Blocker",
   "version": "0.4.6",
 
-  "description": "Protects you from Fraudulent Sites blocking the access. If you access a site that users reported, you'll see how many people reported it on the button on toolbar. The plug-in also allows you to give fraud-feedback on the visited sites through the button on toolbar. Official site on https://fraudblocker.publicvm.com -- For issues and concerns, write to tomasbortoli@gmail.com ",
+  "description": "Protect from Fraudulent Sites blocking the access. Warn on sites reported but not confirmed to be Fraud. You can to report sites too",
+
+  "author": "Tomas Bortoli",
 
   "icons": {
     "48": "icons/icon-48.png",
     "96": "icons/icon-96.png"
   },
-  
-  "applications": {
-    "gecko": {
-      "id": "fraudblocker@mozilla.org",
-      "strict_min_version": "45.0"
-    }
-  },
 
   "permissions": [
     "activeTab",
@@ -29,7 +24,8 @@
   ],  
 
   "background": {
-    "page": "background-page.html"
+    "page": "background-page.html",
+    "persistent": true
   },
 
   "browser_action": {
diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/popup/guiHandler.js ./popup/guiHandler.js
--- ../Fraud-Blocker-multi-browser-extension/popup/guiHandler.js	2017-01-04 12:13:24.847826412 +0100
+++ ./popup/guiHandler.js	2017-03-10 13:48:18.963654966 +0100
@@ -19,6 +19,9 @@
 }
 
 function checkURL(msg){
+  if(msg==null || msg.msg==null)
+    return;
+  
   var done=false;
 
   if(document.getElementById('white').style.display!='none')
diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/popup/popup.css ./popup/popup.css
--- ../Fraud-Blocker-multi-browser-extension/popup/popup.css	2017-01-04 12:13:24.847826412 +0100
+++ ./popup/popup.css	2017-03-10 13:48:18.963654966 +0100
@@ -8,6 +8,7 @@
   user-select: none;           /* Non-prefixed version, currently
                                   not supported by any browser */
   overflow-y: hidden;
+  overflow-x: hidden;
 }
 
 body{
@@ -19,18 +20,22 @@
 }
 
 #black{
+  width: 30em;
   display: none;
 }
 
 #white{
+  width: 30em;
   display: none;
 }
 
 #grey{
+  width: 30em;
   display: none;
 }
 
 #default{
+  width: 24em;
   visibility: hidden;
 }
 
@@ -96,6 +101,7 @@
 }
 
 #thanks{
+  width: 30em;
   display: none;
   padding-top: 5%;
   text-align: center;
@@ -104,12 +110,14 @@
 }
 
 #already{
+  width: 30em;
   display: none;
   text-align: center;
   font-size: 1.1em;
 }
 
 #offline{
+  width: 30em;
   display: none;
   font-size: 1.2em;
 }
diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/popup/popup.html ./popup/popup.html
--- ../Fraud-Blocker-multi-browser-extension/popup/popup.html	2017-01-04 12:13:24.847826412 +0100
+++ ./popup/popup.html	2017-03-10 13:48:18.963654966 +0100
@@ -29,7 +29,7 @@
   </div>
   <div id="white" style="padding-left:10%;padding-right:10%;">
     <img src="/icons/smile.png" id="smile" width="100px" height="100px" />
-    <div class="msg" style="margin-top:5%;">This site is known to be non Fraudulent</div>
+    <div class="msg" style="padding-top:9%;padding-left:3%;">This site is known to be non Fraudulent</div>
   </div>
   <div id="default">
     <img src="/icons/burglar.png" id="burglar" width="100px" height="100px"/>
