diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/background.js ./background.js
--- ../Fraud-Blocker-multi-browser-extension/background.js	2016-12-27 15:58:07.095833918 +0100
+++ ./background.js	2016-12-27 15:41:04.447843238 +0100
@@ -3,16 +3,18 @@
 
 //setup handler to update browser action's badge text
 chrome.tabs.onActivated.addListener(function(activeInfo){
-  chrome.tabs.executeScript(null,{file:"content_script.js"});
-  
   chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
-    var res = getStatus(tabs[0].url);
-    if(res.msg=="grey"){
-      var nFraud=res.nFraud;
-      if(isNaN(nFraud)) //sanitize
-        nFraud='error';
-      chrome.browserAction.setBadgeText({tabId: activeInfo.tabId, text: nFraud});
-      chrome.browserAction.setBadgeBackgroundColor({tabId: activeInfo.tabId, color: "#cc0000"});
+    if(tabs[0].url.substring(0,9)!="chrome://"){
+      chrome.tabs.executeScript(null,{file:"content_script.js"});
+      
+      var res = getStatus(tabs[0].url);
+      if(res.msg=="grey"){
+        var nFraud=res.nFraud;
+        if(isNaN(nFraud)) //sanitize
+          nFraud='error';
+        chrome.browserAction.setBadgeText({tabId: activeInfo.tabId, text: nFraud});
+        chrome.browserAction.setBadgeBackgroundColor({tabId: activeInfo.tabId, color: "#cc0000"});
+      }
     }
   });
 });
diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/manifest.json ./manifest.json
--- ../Fraud-Blocker-multi-browser-extension/manifest.json	2016-12-27 15:58:07.099833918 +0100
+++ ./manifest.json	2016-12-27 15:21:03.083854187 +0100
@@ -4,19 +4,14 @@
   "name": "Fraud Blocker",
   "version": "0.4.2",
 
-  "description": "Protects you from Fraudulent Sites blocking the access. If you access a site that users reported, you'll get a Warning toolbar (can be removed) that will tell you how many people reported the site and how many contro-reported. Also allow you to give fraud-feedback on the visited sites through the button on the toolbar. Official site on https://fraudblocker.publicvm.com -- For issues and concerns, write to tomasbortoli@gmail.com ",
+  "description": "Protect from Fraudulent Sites blocking the access. Warn on sites reported but not confirmed to be Fraud. You can to report sites too",
+
+  "author": "Tomas Bortoli",
 
   "icons": {
     "48": "icons/icon-48.png",
     "96": "icons/icon-96.png"
   },
-  
-  "applications": {
-    "gecko": {
-      "id": "fraudblocker@mozilla.org",
-      "strict_min_version": "45.0"
-    }
-  },
 
   "permissions": [
     "activeTab",
@@ -29,7 +24,8 @@
   ],  
 
   "background": {
-    "page": "background-page.html"
+    "page": "background-page.html",
+    "persistent": true
   },
 
   "browser_action": {
diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/popup/guiHandler.js ./popup/guiHandler.js
--- ../Fraud-Blocker-multi-browser-extension/popup/guiHandler.js	2016-12-27 15:58:07.099833918 +0100
+++ ./popup/guiHandler.js	2016-12-27 15:21:03.083854187 +0100
@@ -19,6 +19,9 @@
 }
 
 function checkURL(msg){
+  if(msg==null || msg.msg==null)
+    return;
+  
   var done=false;
 
   if(document.getElementById('white').style.display!='none')
diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/popup/popup.css ./popup/popup.css
--- ../Fraud-Blocker-multi-browser-extension/popup/popup.css	2016-12-27 15:58:07.099833918 +0100
+++ ./popup/popup.css	2016-12-27 15:37:28.603845205 +0100
@@ -8,6 +8,7 @@
   user-select: none;           /* Non-prefixed version, currently
                                   not supported by any browser */
   overflow-y: hidden;
+  overflow-x: hidden;
 }
 
 body{
@@ -19,18 +20,22 @@
 }
 
 #black{
+  width: 30em;
   display: none;
 }
 
 #white{
+  width: 30em;
   display: none;
 }
 
 #grey{
+  width: 30em;
   display: none;
 }
 
 #default{
+  width: 24em;
   visibility: hidden;
 }
 
@@ -96,6 +101,7 @@
 }
 
 #thanks{
+  width: 30em;
   display: none;
   padding-top: 5%;
   text-align: center;
@@ -104,12 +110,14 @@
 }
 
 #already{
+  width: 30em;
   display: none;
   text-align: center;
   font-size: 1.1em;
 }
 
 #offline{
+  width: 30em;
   display: none;
   font-size: 1.2em;
 }
diff -ur '--exclude=.git' '--exclude=chrome.patch' ../Fraud-Blocker-multi-browser-extension/popup/popup.html ./popup/popup.html
--- ../Fraud-Blocker-multi-browser-extension/popup/popup.html	2016-12-27 15:58:07.099833918 +0100
+++ ./popup/popup.html	2016-12-27 15:21:03.083854187 +0100
@@ -29,7 +29,7 @@
   </div>
   <div id="white" style="padding-left:10%;padding-right:10%;">
     <img src="/icons/smile.png" id="smile" width="100px" height="100px" />
-    <div class="msg" style="margin-top:5%;">This site is known to be non Fraudulent</div>
+    <div class="msg" style="padding-top:9%;padding-left:3%;">This site is known to be non Fraudulent</div>
   </div>
   <div id="default">
     <img src="/icons/burglar.png" id="burglar" width="100px" height="100px"/>
